.github/workflows/release.yaml

```diff
@@ -48,12 +48,13 @@ jobs:
         fetch-depth: 0
 
     - name: "Get tag name"
-      id: git_tag
-      run: |
-        echo ::set-output name=NAME::${GITHUB_REF/refs\/tags\//}
+      run: | # Trim "v" prefix in the release tag
+        RELEASE_TAG=${GITHUB_REF#refs/*/}
+        [[ $RELEASE_TAG = v* ]] && RELEASE_TAG=${RELEASE_TAG:1}
+        echo "RELEASE_VERSION=${RELEASE_TAG}" >> $GITHUB_ENV
```

Comment:
Let's not override the `RELEASE_TAG` value (because the next developer updating release pipeline might need this value unmodified).

Let's change this code to something like
```shell
if [[ "${RELEASE_TAG}" = v* ]]; then RELEASE_VERSION="${RELEASE_TAG:1}" else RELEASE_VERSION="${RELEASE_TAG}" fi
echo "RELEASE_VERSION=${RELEASE_VERSION}" >> $GITHUB_ENV
```